packages/dev-utils/src/constants.js

```diff
@@ -20,10 +20,30 @@ const CRS = [
  *
  * @module params
  */
-module.exports = {
+const constants = {
```

Comment:
we should split out `constants` into objects with more descriptive names. E.g. `signatureConstants`. This shouldn't hold up the merge but we should make a separate issue for this

Reply:
Created an issue for this: https://github.com/AztecProtocol/AZTEC/issues/72